tools/common/__init__.py

```diff
@@ -104,11 +104,11 @@ def print_block_device(device: BlockDevice) -> None:
             info = block.info(attr)
 
             if value is None:
-                value = "-"  # noqa: PLW2901
+                _value = value if value is not None else "-"
```

Comment:
That was my bad, we don't need the first `if` with the 2nd.

Reply:
Oh I'm blind